python/ray/experimental/sgd/pytorch/test/pytorch_distributed_sgd_test.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+import torch
+
+import ray
+
+from ray.experimental.sgd.pytorch.utils import Resources
+from ray.experimental.sgd.pytorch.pytorch_distributed_sgd import PyTorchDistributedSGD
+from ray.experimental.sgd.pytorch.test.test_utils import model_creator, data_creator, optimizer_creator
+
+ray.init()
+
+sgd1 = PyTorchDistributedSGD(model_creator,
+                             data_creator,
+                             optimizer_creator,
+                             num_replicas=3)
+print(sgd1.train())
+sgd1.save("/tmp/distributed_pytorch_checkpoint")
```

Comment:
Maybe use `tempfile` to get the appropriate temp directory (and probably remove the file at the end of the test).

Reply:
or use a with block and `delete=True`